apps/web/src/modules/create-dao/stores/useFormStore.ts

```diff
@@ -147,7 +134,7 @@ export const useFormStore = create(
       resetForm: () => set({ ...initialState }),
     }),
     {
-      name: `nouns-builder-create-${process.env.NEXT_PUBLIC_CHAIN_ID}`,
+      name: `nouns-builder-create-${process.env.NEXT_PUBLIC_NETWORK_TYPE}`,
```

Comment:
just a thought: would we want this to be stored by chain any chance? 🤔

Reply:
the can choose to change the network at any point in the creation flow via the network switched in the nav. would need to rework how we handle networks in the chain flow if we wanted to save each deploy by network. could be added in a followup but will think more on it